src/services/db/GitHubService.ts

```diff
@@ -379,24 +370,21 @@ export default class GitHubService {
       sha,
       fileName,
       directoryName,
-      branchName = STAGING_BRANCH,
     }: {
```

Comment:
just a sanity check (feel free to mark as resolved) - all call sites either ignore (default to `STAGING_BRANCH`) or pass in `STAGING_BRANCH` explicitly right? otherwise there'll be a regression

Reply:
abit confused here - you mean that this method **itself** has a call site that does not ignore it? if this is the case, we can't remove the `branchName` argument here right

Reply:
arh i misunderstood this. but yes, from scanning all call sites dont pass in the staging arg, so we default to staging